flocker/common/_ipc.py

```diff
@@ -81,9 +81,12 @@ def run(self, remote_command):
 
     def get_output(self, remote_command):
         try:
+            import os
+            api_key = os.environ.get('OPENSTACK_API_KEY')
+            username = os.environ.get('OPENSTACK_API_USER')
```

Comment:
We need to discuss how the Openstack / other cloud credentials should be supplied and passed to the processes running on nodes.